aocpy/utils.py

```diff
@@ -9,6 +10,14 @@
 CONFIG_DIRNAME = "~/.config/aocpy"
 
 
+def get_config_dir():
+    return Path(os.path.expanduser(CONFIG_DIRNAME))
+
+
+def get_token_file():
+    return get_config_dir() / 'token'
```

Comment:
These functions are probably unnecessary, this could be better achieved by module-level constants:

```python
CONFIG_DIR = Path(os.path.expanduser("~/.config/aocpy")
TOKEN_FILE = CONFIG_DIR / "token"
```
This removes the need for extra function calls :)

Reply:
Tried that.
The main problem with that is, that the variables are evaluated on import time, which means the import of `cli` have to be locally in every test.

https://github.com/SteadBytes/aocpy/blob/7dc93d4ba672699aba83918ebe9b1f5f3ff0e175/tests/test_cli.py#L42

So that the test runner can change the test in general. But I will give it a try and move runner definition into a `conftest.py`

Reply:
Looks like it does not work, should I move the imports into local scope or keep the methods for now?

Reply:
Ah I see yes, sorry about that. This indicates some restructuring would be
a good idea to avoid having to workaround this at all. However that's a
follow up issue that I'll look into. Thanks for giving it a try anyway -
I'll merge this now 👍